db/memtable.cc

```diff
@@ -117,10 +118,12 @@ MemTable::MemTable(const InternalKeyComparator& cmp,
   // use bloom_filter_ for both whole key and prefix bloom filter
   if ((prefix_extractor_ || moptions_.memtable_whole_key_filtering) &&
       moptions_.memtable_prefix_bloom_bits > 0) {
+    uint32_t bf_bits =
+        filter_bits ? *filter_bits : moptions_.memtable_prefix_bloom_bits;
```

Comment:
Is it possible for *filter_bits = 0?  Should that be guarded/tested against?

Reply:
I changed the logic, now we should be protected against this situation of 0 filter bits.